### ISSUE TITLE
Account Page Serial Number Links / Unsupported Device

### ISSUE THREAD

**Issue by aprsandrew** [aprsworld/mybergey#4]
On the  http://mybergey.aprsworld.com/account/  page, all of the serial number links are forced to be:
http://mybergey.aprsworld.com/serialNumber

For example, serial number A2670 is a device under a customer's account, but is not a ps2tap.  But it forces the URL to be:
http://mybergey.aprsworld.com/A2670
Which it reports is an "unsupported device", and does not go to the appropriate web page for that device.  

A better solution may be for it to fall back on the urlPrimary in the worldData.deviceInfo table when the device is not a PS2Tap.  

In other words, when the device is a ps2tap, it will link to:
http://mybergey.aprsworld.com/serialNumber

But when it is not a ps2tap, it will link to the url listed in the deviceInfo table (which may be a data.aprsworld.com web page).
